webapp/src/app/admin/contacts/contacts.component.ts

```diff
@@ -62,10 +62,22 @@ export class ContactsComponent implements OnInit {
         await this.updateGroups();
     }
 
-    async createContact(view?: string) {
-        const creds: IUserCred = {alias: "", email: "", view: "default", groups: []};
-        const groups = await this.uData.contact.getGroups();
-        creds.groups = groups.map((group) => group.darc.description.toString());
+    /**
+     * Pops up a dialog and asks for the alias and email of the new user to be created.
+     * If any of the view or groups parameter is the non-default, it will not be asked
+     * by the dialog.
+     *
+     * @param view - must be either "" or one of Data.views
```

Comment:
`view` has two meanings, either the name of the view to use (`!== ""`), or letting the user decide (`=== ""`). IMO, it would be best represented by the `string | undefined` type (and also forcing to unwrap it to be typesafe, as `Option` in functional languages)